testData/solutions/win32.json5

```diff
@@ -3025,6 +3025,614 @@
+                            "description": "Standard currency for being used in the system",
+                            "type": "string",
+                            "enum": ["$", "€"]
+                        }
+                    },
+                    "iCurrency": {
+                        "schema": {
+                            "title": "Positive currency format",
+                            "description": "Format in which express money cuantities",
+                            "type": "number",
+                            "range": "0..3",
+                            "outputFormat": ["$1.1", "1.1$", "$ 1.1", "1.1 $"]
+                        }
+                    },
+                    "iNegCurr": {
+                        "schema": {
+                            "title": "Negative currency format",
+                            "description": "Format in which express negative money cuantities",
+                            "type": "number",
+                            "range": "0..5",
+                            "outputFormat": ["($1.1)", "-$1.1", "$-1.1", "$1.1-", "($1.1)", "-1.1$"]
+                        }
+                    },
+                    "sMonDecimalSep": {
+                        "schema": {
+                            "title": "Currency decimal symbol",
+                            "description": "The symbol for decimal in monetary amount",
+                            "type": "string with a max of three chars",
+                            "defaults": [",", ".", "'"]
+                        }
+                    },
+                    "iCurrDigits": {
+                        "schema": {
+                            "title": "Currency decimal digits",
+                            "description": "Number of decimal digits in a monetary amount",
+                            "type": "number",
+                            "range": "0..9"
+                        }
+                    },
+                    "sMonThousandSep": {
+                        "schema": {
+                            "title": "Currency digits grouping symbol",
+                            "description": "Determines the grouping symbol for digits in a monetary amount representation",
+                            "type": "string",
+                            "enum": [",", ".", "'"]
+                        }
+                    },
+                    "sMonGrouping": {
+                        "schema": {
+                            "title": "Currency digits grouping format",
+                            "description": "Determines the grouping format for digits in a monetary amount representation",
+                            "type": "string",
+                            "enum": ["0;0", "3;0", "3;0;0", "3;2;0"],
+                            "outputFormat": ["123457$", "123456,789$", "123,456,79$", "12,34,56,789$"]
+                        }
+                    }
+                },
+                // FUTURE:
```

Comment:
This huge comment should probably be put in another file or else just pasted into a JIRA